### PR TITLE
CLI: Exclude resource types

### PR DESCRIPTION
**Pull request by AleF83**

### CODE REVIEW

File: cli/oclif.manifest.json [hunk 1]
```diff
@@ -57,7 +57,8 @@
           "type": "option",
           "description": "Custom authorization header",
           "required": false
-        }
+        },
+        "exclude": { "name": "exclude", "type": "option", "description": "Resource types to skip", "required": false }
```
Comment (tomeresk) [inline cli/oclif.manifest.json:61]: I think `exclude` is too general, maybe `exclude-resource-types` or `ignore-resource-types`?

Also description should specify how to send multiple types, e.g. `A comma separated list of resource types to skip/ignore` (skip or ignore based on chosen name)

There is also an option that allows setting a flag multiple times (`multiple: true` https://oclif.io/docs/flags), not sure how the output is presented based on the docs but worth checking

File: cli/tests/apply-resources.spec.ts [hunk 2]
```diff
@@ -12,10 +12,17 @@ describe('Apply resources', () => {
       })
     )
     .stdout()
-    .command(['apply:resources', '--dry-run', '--registry-url=http://registry/graphql', 'tests/resources'])
+    .command([
+      'apply:resources',
+      '--dry-run',
+      '--registry-url=http://registry/graphql',
+      '--exclude=upstreams',
```
Comment (tomeresk) [inline cli/tests/apply-resources.spec.ts:19]: Should also add a test for skipping multiple resource types

**Review by tomeresk — CHANGES_REQUESTED** [review]

File: cli/oclif.manifest.json (hunk 1 above)
Comment (AleF83) [reply cli/oclif.manifest.json:61]: Renamed to `skip-resource-types`.
I don't think that this is the case of `multiple` flag. It's more suitable for something like `-f` in `kubeclt` command.

**Review by tomeresk — APPROVED** [review]
Still missing the mention of the comma separated list in the description

**Pull request merged**